### PR TITLE
Piler + Mreps recipes 

### PR DESCRIPTION
**Pull request by oinizan**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This PR add 2 new recipes so I mention @bioconda/core to get a review.

### CODE REVIEW

File: recipes/piler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  build:
+  - gcc # [linux]
+  - llvm # [osx]
+
+  run:
+  - libgcc # [linux]
+
+test:
+  commands:
+    - piler2 -version # [linux]
```
Comment (bgruening) [inline recipes/piler/meta.yaml]: Uhh, why is that?
I don't think it is good to have different behaviour for different OS. Can we symlink the one to the other and make it consistent?

File: recipes/mreps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package:
+  name: mreps
+  version: "0.1"
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/gregorykucherov/mreps.git
```
Comment (bgruening) [inline recipes/mreps/meta.yaml]: Can you add a revision, or better use a tar.gz as download with checksum?

**Comment by oinizan** [comment]
@bgruening For Piler: Instead of a symlink I choose to call the executable piler (and not piler 2, see the sed in the Makefile). It is more consistent with the "real" version of piler (1.0). What do you think ?

File: recipes/mreps/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  url: http://mreps.univ-mlv.fr/mreps-2.6.tar
+  md5: 4cc5444676501adeddf09f0aaf68dde8
+
+requirements:
+  build:
+  - gcc # [linux]
+  - llvm # [osx]
+
+  run:
+  - libgcc
```
Comment (bgruening) [inline recipes/mreps/meta.yaml]: only linux

File: recipes/mreps/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  files:
+    - DmelChr4.fa
+  commands:
+    - mreps -res 3 -exp 3.0 -from 10000 -to 12000 DmelChr4.fa
+
+about:
+  home: http://mreps.univ-mlv.fr/ 
+  license: GNU GENERAL PUBLIC LICENSE
+  license_file: LICENSE.md
+  summary: GNU GENERAL PUBLIC LICENSE Version 2, June 1991
```
Comment (bgruening) [inline recipes/mreps/meta.yaml]: This is the tool summary not the license :)

**Comment by bgruening** [comment]
@oinizan this is a great fix! Thanks! This fix should also go upstream I think.

**Comment by oinizan** [comment]
@bgruening let me know ...

**Comment by bgruening** [comment]
Great! Thanks!

**Pull request merged**